express-hello-world/server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res) => {
+  console.log(req.method);
+  console.log(res.send('Hello Robert'));
```

Comment:
Don't log the `res.send` call itself. You can, but you should treat console.log calls as if they are optional. If someone deleted all the log messages, the code should still work.

I appreciate the greeting! :-)